libtransmission/web.c

```diff
@@ -183,7 +183,6 @@ static CURL* createEasy(tr_session* s, struct tr_web* web, struct tr_web_task* t
     task->timeout_secs = getTimeoutFromURL(task);
 
     curl_easy_setopt(e, CURLOPT_AUTOREFERER, 1L);
-    curl_easy_setopt(e, CURLOPT_ENCODING, "gzip;q=1.0, deflate, identity");
```

Comment:
Instead of unconditionally refusing to accept compressed responses, I'd rather check whether [curl_version_info()](https://curl.haxx.se/libcurl/c/curl_version_info.html) indicates support for `CURL_VERSION_LIBZ` feature. Obsiously, not on each and every request :)